### PR TITLE
Various optimizations

### PR DESCRIPTION
**Pull request by freimair**
Various optimizations to the Bisq code
- reduce info to debug log on spamming infos
- do not do parsing until everything is completed and thus, save on resources and time

### CODE REVIEW

File: core/src/main/java/bisq/core/dao/governance/proposal/ProposalValidator.java [hunk 1]
```diff
@@ -55,7 +55,8 @@ public boolean areDataFieldsValid(Proposal proposal) {
             validateDataFields(proposal);
             return true;
         } catch (ProposalValidationException e) {
-            log.warn("proposal data fields are invalid. proposal={}, error={}", proposal, e.toString());
+            // TODO only log once dao state is complete
+            //log.warn("proposal data fields are invalid. proposal={}, error={}", proposal, e.toString());
```
Comment (ManfredKarrer) [inline core/src/main/java/bisq/core/dao/governance/proposal/ProposalValidator.java]: That should be still used. The fix in the subclass avoids the wrong logs

**Review by ManfredKarrer — CHANGES_REQUESTED** [review]
NACK

See comment

File: core/src/main/java/bisq/core/dao/governance/proposal/ProposalValidator.java (hunk 1 above)
Comment (freimair) [reply core/src/main/java/bisq/core/dao/governance/proposal/ProposalValidator.java]: force-pushed without this very change

Comment (christophsturm) [reply core/src/main/java/bisq/core/dao/governance/proposal/ProposalValidator.java]: where is the fix that avoids that?

Comment (ManfredKarrer) [reply core/src/main/java/bisq/core/dao/governance/proposal/ProposalValidator.java]: Would have been better in a new commit so make it more transparent.

**Review by ManfredKarrer — APPROVED** [review]
utACK

**Review by sqrrm — APPROVED** [review]
ACK

**Pull request merged**